.drone.yml

```diff
@@ -348,17 +348,14 @@ platform:
 
 steps:
 - name: dispatch
-  image: rancher/dapper:v0.4.2
+  image: curlimages/curl:7.69.1
   environment:
     PAT_USERNAME:
       from_secret: pat_username
```

Comment:
I don't know about the drone env but is there any exfiltration risk here?

Reply:
I don't think so, drone hides secret info, we are using the same method in k3s drone as well